lib/colonel_kurtz/renderer.ex

```diff
@@ -20,19 +30,22 @@ defmodule ColonelKurtz.Renderer do
     end
   end
 
+  @spec assigns(block) :: [content: map, children: list(iodata)]
   defp assigns(%{content: content, blocks: blocks} = _block) do
     [
       content: content,
       children: render_blocks(blocks)
     ]
   end
 
+  @spec view_module(binary) :: module
   defp view_module(type) do
     Module.concat(block_views_module(), Macro.camelize(type) <> "View")
   end
 
-  defp block_views_module() do
-    case Application.fetch_env!(:colonel_kurtz_ex, ColonelKurtz) do
+  @spec block_views_module :: module
+  defp block_views_module do
+    case Application.get_env(:colonel_kurtz_ex, ColonelKurtz) do
```

Comment:
`fetch_env/1` fetches values from the app env at compile time and raises if not defined. We're specifically handling undefined config in our `case`, so we want the safe version rather than the one that throws.

When I started implementing tests I found that `fetch_env` yielded nothing while `get_env` gives the expected result. This is something I'd like to figure out the correct answer to. It's possible that we just need to adjust how we're setting up our tests and then we can switch back to `fetch_env/1` (without the bang `!`).